HISTORY.md

```diff
@@ -1,5 +1,32 @@
 # Release History
 
+## 0.3.1 (2020-04-27)
+
+- Adds p2p_stub connection
+- Adds p2p_noise connection
+- Adds webhook connection
+- Upgrade error handling for error skill
+- Fix default timeout on main agent loop and provide seter in AEABuilder
```

Comment:
fix typo: `setter` instead of `seter`

Reply:
should we add an issue to add an option on `aea run`, or also in `aea-config.yaml`?